app/pages/UcsfClientEligibilityPage/UcsfClientEligibilityPage.module.scss

```diff
@@ -0,0 +1,108 @@
+@import '~styles/utils/_helpers.scss';
+
+.eligibilityPage {
+  grid-column: 2 / 13;
+  @media screen and (max-width: 1370px) {
+    grid-column: 2/11;
+  }
+
+  @media screen and (max-width: $break-tablet-p) {
+    grid-column: 2/12;
+  }
+
+  @media screen and (max-width: $break-tablet-s) {
+    grid-column: 1/13;
+  }
+}
+
+.eligibilitiesContainer {
+  display: grid;
+  grid-template-columns: repeat(22, 1fr);
```

Comment:
I used grid 22 for a placing an element in the middle of a column.  (Since this page was taking up an 11 span, I had to use 22 to get the math right). It was a little overcomplicated to manage this though.

I guess the point is moot given we now are using an updated design, and I have removed the 22 column grid!